app/src/main/java/com/framgia/quangtran/music_42/ui/homescreen/adapters/ViewPagerAdapter.java

```diff
@@ -16,16 +18,21 @@
     private static final int TAB_HOME = 0;
     private static final int TAB_PERSONAL = 1;
     private static final int TAB_SETTING = 2;
+    public Bundle mBundle;
 
-    public ViewPagerAdapter(FragmentManager fm) {
+    public ViewPagerAdapter(FragmentManager fm, Bundle bundle) {
         super(fm);
+        mBundle = bundle;
     }
 
     @Override
     public Fragment getItem(int i) {
         switch (i) {
             case TAB_HOME:
-                return new HomeFragment();
+                HomeFragment homeFragment = new HomeFragment();
```

Comment:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment
em nên sử dụng theo convention này

Reply:
Em chưa hiểu lắm em vẫn theo convention đó nhưng bundle này em gửi từ activity sang anh ạ

Reply:
> Em chưa hiểu lắm em vẫn theo convention đó nhưng bundle này em gửi từ activity sang anh ạ

Em tạo một phương thức statatic trả về instance bên HomeFragment của em.